lib/interface_quda.cpp

```diff
@@ -5741,15 +5799,14 @@ double qChargeQuda()
   } else {
     gauge = gaugeSmeared;
   }
-  // Do we keep the smeared extended field on memory, or the unsmeared one?
 
   profileQCharge.TPSTART(QUDA_PROFILE_INIT);
   // create the Fmunu field
 
   GaugeFieldParam tensorParam(gaugePrecise->X(), gauge->Precision(), QUDA_RECONSTRUCT_NO, 0, QUDA_TENSOR_GEOMETRY);
   tensorParam.siteSubset = QUDA_FULL_SITE_SUBSET;
   tensorParam.order = QUDA_FLOAT2_GAUGE_ORDER;
-  tensorParam.ghostExchange = QUDA_GHOST_EXCHANGE_NO;
+  tensorParam.ghostExchange = QUDA_GHOST_EXCHANGE_PAD;
```

Comment:
This is a bug that has been introduced here.  This should remain at `QUDA_GHOST_EXCHANGE_NO`.

FYI: `QUDA_GHOST_EXCHANGE_PAD` is a specific "inline ghost" format used by the stencil operators, but for all pure-gauge algorithms we should only be using either `QUDA_GHOST_EXCHANGE_NO` or `QUDA_GHOST_EXCHANGE_EXTENDED`